src/services/kidvids/index.jsx

```
@@ -0,0 +1,425 @@
+import { Toast } from "../../components/theme/Toast";
+import {
+  addCommentOnKidVids,
+  addKidVids,
+  getKidVidsComments,
+  getKidVidsLikes,
+  getKidVidsSubCategoriesByCategory,
+  likeUnlikeKidVids,
+} from "../../app/features/kidvids";
+
+export const AddKidVids = ({
+  setAddModal,
```

ESLint

[error] 'setAddModal' is missing in props validation

(react/prop-types)